src/lightning_app/components/multi_node.py

```diff
@@ -90,6 +90,7 @@ def run(self) -> None:
             self.ws[node_rank].run(
                 main_address=self.ws[0].internal_ip,
                 main_port=self.ws[0].port,
+                nodes=self.nodes,
```

Comment:
I think a better name would be `num_nodes`.
Also need to update the example in the docstring: `AnyDistributedComponent.run`

Reply:
Done !